modules/nw-enabling-static-egress-ips.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * networking/enabling-static-egress-ips.adoc
+
+[id="manually-assigned-{context}"]
+= Configuring manually-assigned egress IPs
+
+In {product-title} you can associate one or more egress IPs with a project.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster with a user that has the `cluster-admin` role.
+
+.Procedure
+
+. Update the `NetNamespace` by specifying the following JSON
+object with the desired IP addresses:
++
+----
+$ oc patch netnamespace <project> -p \ <1>
+  '{
+    "egressIPs": [ <2>
+      "<ip_address>"
+      ]
+  }'
+----
+<1> Specify the name of the project.
+<2> Specify one or more egress IP addresses. The `egressIPs` field is an array.
++
+For example, to assign the `project1` project to an IP address of
+`192.168.1.100`:
++
+----
+$ oc patch netnamespace project1 -p '{"egressIPs": ["192.168.1.100"]}'
+----
++
+The `egressIPs` field is an array. You can set `egressIPs`
+to two or more IP addresses on different nodes to provide high
+availability. If multiple egress IP addresses are set, pods use the first IP in
+the list for egress, but if the node hosting that IP address fails, pods
+switch to using the next IP in the list after a short delay.
+
+. Manually assign the egress IP to the desired node hosts. Set the `egressIPs`
+field on the `HostSubnet` object on the node host. Using the following JSON,
+include as many IPs as you want to assign to that node host:
++
+----
+$ oc patch hostsubnet <node_name> -p \ <1>
+  '{
+    "egressIPs": [ <2>
+      "<ip_address_1>",
+      "<ip_address_N>"
+      ]
+  }'
+----
+<1> Specify the name of the project.
+<2> Specify one or more egress IP addresses. The `egressIPs` field is an array.
++
+For example, to say that `node1` should have the egress IPs `192.168.1.100`,
+`192.168.1.101`, and `192.168.1.102`:
++
+----
+$ oc patch hostsubnet node1 -p \
+  '{"egressIPs": ["192.168.1.100", "192.168.1.101", "192.168.1.102"]}'
+----
++
+[IMPORTANT]
+====
+Egress IPs are implemented as additional IP addresses on the primary network
+interface, and must be in the same subnet as the node's primary IP.
+Additionally, any external IPs should not be configured in any Linux network
+configuration files, such as *_ifcfg-eth0_*.
```

Comment:
In 4.x docs, file names will be marked in backticks per https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference.